<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-10-25T08:26:42 CEST" name="Suite" finished-at="2023-10-25T08:27:44 CEST" duration-ms="61478">
    <groups>
    </groups>
    <test started-at="2023-10-25T08:26:42 CEST" name="Test" finished-at="2023-10-25T08:27:44 CEST" duration-ms="61478">
      <class name="testNGTests.EnglishAboutUs">
        <test-method is-config="true" signature="open()[pri:0, instance:testNGTests.EnglishAboutUs@28dcca0c]" started-at="2023-10-25T08:26:42 CEST" name="open" finished-at="2023-10-25T08:26:47 CEST" duration-ms="4353" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- open -->
        <test-method signature="EnAboutUs()[pri:0, instance:testNGTests.EnglishAboutUs@28dcca0c]" started-at="2023-10-25T08:26:47 CEST" name="EnAboutUs" finished-at="2023-10-25T08:26:48 CEST" duration-ms="1662" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EnAboutUs -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testNGTests.EnglishAboutUs@28dcca0c]" started-at="2023-10-25T08:26:48 CEST" name="tearDown" finished-at="2023-10-25T08:26:49 CEST" duration-ms="200" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testNGTests.EnglishAboutUs -->
      <class name="testNGTests.GermanContact">
        <test-method is-config="true" signature="open()[pri:0, instance:testNGTests.GermanContact@57bc27f5]" started-at="2023-10-25T08:27:16 CEST" name="open" finished-at="2023-10-25T08:27:17 CEST" duration-ms="1070" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- open -->
        <test-method signature="DeContact()[pri:0, instance:testNGTests.GermanContact@57bc27f5]" started-at="2023-10-25T08:27:17 CEST" name="DeContact" finished-at="2023-10-25T08:27:44 CEST" duration-ms="26240" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Proxy element for: DefaultElementLocator 'By.xpath: //h1[normalize-space()='Wir freuen uns 端ber Ihre Kontaktaufnahme']'] but found [null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Proxy element for: DefaultElementLocator 'By.xpath: //h1[normalize-space()='Wir freuen uns 端ber Ihre Kontaktaufnahme']'] but found [null]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:617)
at testNGTests.GermanContact.DeContact(GermanContact.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DeContact -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testNGTests.GermanContact@57bc27f5]" started-at="2023-10-25T08:27:44 CEST" name="tearDown" finished-at="2023-10-25T08:27:44 CEST" duration-ms="189" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testNGTests.GermanContact -->
      <class name="testNGTests.GermanAboutUs">
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:testNGTests.GermanAboutUs@5c7933ad]" started-at="2023-10-25T08:26:49 CEST" name="openBrowser" finished-at="2023-10-25T08:26:51 CEST" duration-ms="2374" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="DeAboutUs()[pri:0, instance:testNGTests.GermanAboutUs@5c7933ad]" started-at="2023-10-25T08:26:51 CEST" name="DeAboutUs" finished-at="2023-10-25T08:27:16 CEST" duration-ms="24841" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Proxy element for: DefaultElementLocator 'By.xpath: //h1[normalize-space()='Wir freuen uns 端ber Ihre Kontaktaufnahme']'] but found [null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Proxy element for: DefaultElementLocator 'By.xpath: //h1[normalize-space()='Wir freuen uns 端ber Ihre Kontaktaufnahme']'] but found [null]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:617)
at testNGTests.GermanAboutUs.DeAboutUs(GermanAboutUs.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DeAboutUs -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testNGTests.GermanAboutUs@5c7933ad]" started-at="2023-10-25T08:27:16 CEST" name="tearDown" finished-at="2023-10-25T08:27:16 CEST" duration-ms="192" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testNGTests.GermanAboutUs -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
